pandas/core/arrays/base.py

```diff
@@ -878,6 +885,21 @@ def interpolate(
     ) -> Self:
         """
         See DataFrame.interpolate.__doc__.
+
+        Examples
+        --------
+        >>> arr = pd.arrays.NumpyExtensionArray(np.array([0, 1, np.nan, 3]))
+        >>> arr.interpolate(method="linear",
+        ...                 limit=3,
+        ...                 limit_direction="forward",
+        ...                 index=pd.Index([1, 2, 3, 4]),
+        ...                 fill_value=1,
+        ...                 copy=False, axis=0,
```

Comment:
Nit: Could you put `axis` on a new line?

Reply:
Sure